src/plot_utils.py

```diff
@@ -79,7 +79,7 @@ def mi_hist(bdb, generator, col1, col2, num_samples=1000, bins=5):
 
 
 def heatmap(bdb, bql, vmin=None, vmax=None, row_ordering=None,
-        col_ordering=None):
+            col_ordering=None):
```

Comment:
The preferred convention in `bdbcontrib` and `bayeslite` to keep indenting until no longer necessary (i.e. here we just need two indents so that `col_ordering` does not overlap with the docstring and the function code), rather than aligning the next line with the opening parenthesis.

I made several such whitespaces changes when refactored `bdbcontrib` to regularize the codebase.